entrypoint.sh

```diff
@@ -5,13 +5,17 @@ set -e
 # setup the environment
 if [ "$1" = 'SickBeard.py' ]; then
   if [ ! -f ${APP_DATA}/config.ini ]; then
-    cp /template/config.ini ${APP_DATA}/
+    exec gosu ${APP_UID:=0}:${APP_GID:=0} cp /template/config.ini ${APP_DATA}/
+  fi
+  CFG_DIR_UID=`ls -lnd "$APP_DATA" | awk 'NR==1 {print $3}'`
+  CFG_DIR_GID=`ls -lnd "$APP_DATA" | awk 'NR==1 {print $4}'`
+  if [ $APP_UID -ne $CFG_DIR_UID ] && [ $APP_GID -ne $CFG_DIR_GID ]; then
```

Comment:
That is a good point. At least busybox doesn't support format strings properly in stat, but considering that the image is built on Debian, we should be fine with using stat.

Also considering that we depend on external utilities anyway and we can't really run on a busybox environment, this should be a safe call in terms of portability as well.